src/applications/vaos/project-cheetah/components/SecondDosePage.jsx

```diff
@@ -0,0 +1,119 @@
+import React, { useEffect, useState } from 'react';
+import { useHistory } from 'react-router-dom';
+import { connect } from 'react-redux';
+import * as actions from '../redux/actions';
+import FormButtons from '../../components/FormButtons';
+import { scrollAndFocus } from '../../utils/scrollAndFocus';
+import moment from 'moment';
+import { getReviewPage } from '../redux/selectors';
+import AdditionalInfoRow from '../../appointment-list/components/cards/AdditionalInfoRow';
+
+const pageKey = 'secondDosePage';
+const pageTitle = 'Plan your second dose';
+
+function goBack({ routeToPreviousAppointmentPage, history }) {
+  return routeToPreviousAppointmentPage(history, pageKey);
+}
+
+function SecondDosePage({
+  data,
+  facility,
+  pageChangeInProgress,
+  routeToNextAppointmentPage,
+  routeToPreviousAppointmentPage,
+}) {
+  const [showMoreOpen, setShowMoreOpen] = useState(false);
+  const history = useHistory();
+  const { date1, vaFacility } = data;
+
+  useEffect(() => {
+    if (history && !vaFacility) {
+      history.replace('/new-project-cheetah-booking');
+    }
+    document.title = `${pageTitle} | Veterans Affairs`;
+    scrollAndFocus();
+  }, []);
+
+  return (
+    <div>
+      <h1>{pageTitle}</h1>
+      <div>
+        <p>
+          You'll need to return to the {facility.name} for your second dose.
+          Your team will schedule your second dose after you receive you first.
+        </p>
+        <p>
+          If you receive your first dose on{' '}
+          <strong>
+            {moment(date1, 'YYYY-MM-DDTHH:mm:ssZ').format(
+              'dddd, MMMM DD, YYYY ',
+            )}{' '}
+          </strong>
+          and receive:
+        </p>
+        <h2 className="vads-u-font-size--base vads-u-font-family--sans vads-u-margin-bottom--0">
+          Moderna
+        </h2>
+        <div className="vads-u-padding-bottom--1">
+          Return for your second dose after{' '}
+          {moment(date1, 'YYYY-MM-DDTHH:mm:ssZ')
+            .add(28, 'days')
+            .format('dddd, MMMM DD, YYYY')}
+        </div>
+        <h2 className="vads-u-font-size--base vads-u-font-family--sans vads-u-margin-bottom--0 vaos-appts__block-label">
+          Pfizer
+        </h2>
+        <div>
+          Return for your second dose after{' '}
+          {moment(date1, 'YYYY-MM-DDTHH:mm:ssZ')
+            .add(21, 'days')
+            .format('dddd, MMMM DD, YYYY')}
+        </div>
+      </div>
+      <div className="vads-u-margin-y--4">
+        <AdditionalInfoRow
```

Comment:
I think it would be better to use the AdditionalInfo component from the design system: https://design.va.gov/storybook/?path=/docs/components-additionalinfo--default

This local one we have is going to be deleted when we migrate to the new homepage, hopefully.

Reply:
done